Assets/BossRoom/Scripts/Server/Game/Entity/ServerDisplacerOnParentChange.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections;
+using Unity.Netcode;
+using Unity.Netcode.Components;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    /// <summary>
+    /// Component to simply play a raising/descending animation when this NetworkObject's parent NetworkObject changes.
+    /// </summary>
+    /// <remarks>
+    /// Currently, Netcode for GameObjects' (Netcode) NetworkAnimator component does not support animations that apply
+    /// Root Motion. This script is a workaround and will be refactored when Root Motion-based animations are supported.
+    /// </remarks>
+    public class ServerDisplacerOnParentChange : NetworkBehaviour
```

Comment:
So this class is basically to lerp the object while the animation is running. I'm wondering if there's anything in NetworkTransform's interpolation that's messing with this.